src/DeVinci_frontend/components/SelectModel.svelte

```diff
@@ -54,6 +55,7 @@
             size={option.size}
             bind:chatModelDownloadInProgress
           />
+          <Entertainment />
```

Comment:
it didn't show up on the screen for me; 
I think it should be placed outside the loop (/each) and wrapped within an if statement 
{#if chatModelDownloadInProgress}
such that it shows up whenever a model is being downloaded